kaitai_utils.py

```diff
@@ -37,6 +43,35 @@ def _get_metadata():
     with open(_get_ksy_file(), 'r') as file:
         return yaml.safe_load(file)['meta']
 
+def token_has_process2(object):
+    if type(object) is dict:
+        for key in object.keys():
+            return 'process' in object or token_has_process2(object[key])
+    elif type(object) is list:
+        for item in object:
+            return False or token_has_process2(item)
+    else:
+
+
+
+def token_has_process(dictionary: dict):
```

Comment:
Private methods are all prefixed. As this is private, please prefix private methods with a `_`